ttg/ttg/util/env.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Created by Eduard Valeyev on 11/6/21.
```

Comment:
Do we have proper copyright headers?

Reply:
not that I know of ... we should discuss